app.js

```diff
@@ -31,7 +31,7 @@ app.use((req, res, next) => {
   var ip = req.headers["x-forwarded-for"];
   var agent = req.headers["user-agent"];
   console.log("IP:", ip, "AGENT:", agent);
-  if(BLOCKLIST.includes(ip) || BLOCKAGENTS.includes(agent)) {
+  if(BLOCKLIST.includes(ip) || BLOCKAGENTS.includes(agent) || agent.includes("Presto")) {
```

Comment:
couldn't we have just added ",Presto" to the BLOCKAGENTS env var? https://github.com/socrata/opendatanetwork.com/blob/e5ce4ed1ec06b282c80d572b849fb7502803ac78/app.js#L29

Reply:
Sadly that is a full text match and the user agent is not completely static